wp-content/themes/core/pcss/base/layouts/_split.pcss

```diff
@@ -0,0 +1,113 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Layout: Split
+ *
+ * Usage: Panel with Content Block and Image Block (50/50, 60/40, etc.)
+ *
+ * Example:
+ * <div class="l-split-image-right">
+ *   <div class="split__image">
+ *     ...
+ *   </div>
+  *  <div class="split__content">
+ *     ...
+ *   </div>
+ * </div>
+ *
+ * ----------------------------------------------------------------------------- */
+
+/* Layout: Split Image Right - 50/50 */
+
+.l-split-image-right {
```

Comment:
Solid first pass on this, let's take it just a step further. I'd recommend creating a `l-split-image` class for which all the shared styles can live, and then a `l-split-image--right` and `l-split-image--left`. If you can address that, kick your ticket over to me and I'll get this wrapped up tomorrow. I want to think through some additional image component utility classes 🙇